index.js

```diff
@@ -922,19 +913,21 @@ class Watcher {
   }
 
   async _waitForChanges () {
-    if (this.current.version < this.bee.version || this.closed) return
+    if (this.current.version < this.bee.version || this.closing) return
 
     await new Promise(resolve => {
       this._resolveOnChange = resolve
     })
   }
 
   async next () {
+    if (!this.opened) await this.ready()
```

Comment:
Can you link which test passes with this addition?

That is already in here:
https://github.com/holepunchto/hyperbee/blob/3595f1709c29964bd706357310fd66f71f23fe27/index.js#L941
So it looks unnecessary

Reply:
Ah, you're right, well spotted! With the other changes I made it no longer fails if I remove that line, so whatever it was solving back then is no longer relevant

Removed it